addons/project_sms/views/project_task_type_views.xml

```diff
@@ -6,7 +6,8 @@
         <field name="inherit_id" ref="project.task_type_edit"/>
         <field name="arch" type="xml">
             <field name="mail_template_id" position="after">
-                <field name="sms_template_id" context="{'default_model': 'project.task'}" options="{'no_quick_create': True}" invisible="user_id"/>
+                <field name="sms_template_id" placeholder="Don't send text" invisible="user_id"
```

Comment:
Global detail note: I think we usually don't use shortened form, so "Do not send text" instead of "Don't send text", but not sure if it's still the case or not, would keep the product owner version in doubt.

Reply:
cf the other comment